go.mod

```diff
@@ -24,6 +24,4 @@ require (
 
 replace (
```

Comment:
My local go commands replace this with 
```
replace git.apache.org/thrift.git => github.com/apache/thrift v0.0.0-20180902110319-2566ecd5d999
```

Would you please run `go mod tidy` once?

Reply:
done